autoload/lsp/ui/vim.vim

```diff
@@ -239,25 +240,33 @@ function! s:handle_symbol(server, last_req_id, type, data) abort
     endif
 endfunction
 
-function! s:handle_location(server, last_req_id, type, data) abort
+function! s:handle_location(jump_if_one, ctx, server, last_req_id, type, data) abort "ctx = {counter, list}
```

Comment:
Can we move `jump_if_one` to `ctx` too? It is easier to add/remove from `ctx` then change params. Feel free to move `last_req_id` to `ctx` else I can do this in the future.